pkgs/by-name/th/thorium/package.nix

```
@@ -0,0 +1,40 @@
+{
+  stdenv,
+  fetchurl,
+  undmg,
+  lib,
+}:
+stdenv.mkDerivation (finalAttrs: rec {
```

`finalAttrs` and `rec` are not meant to be used together.

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

---

@quinneden Is this marked as resolved by accident? There seems no further changes uploaded nor comments explaining the decision.

---

> @quinneden Is this marked as resolved by accident? There seems no further changes uploaded nor comments explaining the decision.

Yes, my apologies.